Makefile

```diff
@@ -1,11 +1,10 @@
 .PHONY: all build clean
 
-all: clean build 
+all: clean build
 
 build:
-	bash -c "./scripts/build.sh supervisor"
-	bash -c "./scripts/build.sh journald_scale_test"
+	go build ./...
 
 
 clean:
-	rm -rf ./build
```

Comment:
I found it easier to have a single node build tree in CI, knowing I can access all my artifacts from `./build/...`. We might want to add this back later.